### DISCUSSION TITLE
軟體工程師薪資前景差很大？！這三個產業最美好

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#458] [General]
軟體工程師薪資前景差很大？！這三個產業最美好
大家程式要好好學阿哈哈哈
https://www.technice.com.tw/ai/248/

**Comment by brian431** [comment]
> 軟體工程師薪資前景差很大？！這三個產業最美好
> 
> 大家程式要好好學阿哈哈哈
> 
> https://www.technice.com.tw/ai/248/

在台灣還是涉及到硬體的比較賺錢一點。不過我個人對硬體沒什麼興趣，想要在純軟拿到跟硬體一樣的薪資，需要更大程度地提升個人能力。

**Comment by lx3211** [comment]
如果想在軟體業賺大錢，最好的還是去國外發展，台灣就是硬體王國qq

**Comment by Chenszuhong** [comment]
確實程式能力是決定薪資大小的關鍵

**Comment by xCk27x** [comment]
在台灣硬體真的比軟體強很多
軟體頂大概就200多一點
如果希望有更好的發展還是往國外去比較快

**Comment by 32844583** [comment]
軟體確實在台灣不吃香，但還是處決於個人實力

**Comment by BingRong1217** [comment]
現在軟硬體方面好熱門，大家都說軟硬體整合的工程師較吃香

**Comment by limitx0** [comment]
在臺灣做硬體還是比軟體賺錢
雖然軟體工作是比較多

**Comment by awehao** [comment]
擔心這些的同時，也要不斷增進自己的實力，功力不夠學無止境

**Comment by larry40305** [comment]
想走軟體還是必須得出國，在台灣不好發展

**Comment by 710388** [comment]
修昔底德說過一句經典的名言，歷史會重演。這句話令我不禁感慨問題的迫切性。馬克思講過一句值得人反覆尋思的話，思考一切。這句話改變了我的人生。說到薪資 ，你會想到什麼呢？
